addon-templates/kubectl/heapster/heapster-deployment.yaml

```diff
@@ -16,6 +16,6 @@ spec:
         image: $GCR_IO_REGISTRY/google_containers/heapster-amd64:v1.3.0-beta.1
         command:
         - /heapster
-        - --source=kubernetes:https://10.43.0.1:443?inClusterConfig=true&useServiceAccount=true
+        - --source=kubernetes:https://kubernetes:6443?inClusterConfig=true&useServiceAccount=true
```

Comment:
why port change?

Reply:
because heapster will reach the pod ip directly of kubernetes instead of the service ip, i wanted to avoid passing the kubernetes master service ip to the addons container as well, the alternative is to use `kubernetes.default` (which is the service dns name), however this will require adding kubernetes.default to the sans and will break due to the certificate bug even for upgrades that doesn't change the default subnets

Reply:
@galal-hussein I don't think we can do it. It should use kubernetes service ip address. Can you use the variable we configure for san in the template?